code/parsers/wget.py

```diff
@@ -33,29 +34,42 @@ class WgetParser(ParserBase):
 
     def on_load(self) -> None:
         self.wget_block_extractor_regex = re.compile(
-            "\\b(wget (-.*)?(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?[\s\S]+? saved \[\d+/\d+\])"
+            r"\b(wget (-.*)?((https?:\/\/)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&\/=]*)))\s"
         )
-        self.url_extractor = re.compile(
-            "((http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/|ftp:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?)(\n|\r\n)"
+        self.url_extractor_regex = re.compile(
+            r"((http|ftp|https):\/\/([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:\/~+#-]*/([\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])))"
+        )
+        self.output_extractor_regex = re.compile(
+            r"-O (.*?) "
         )
-        self.ip_extractor = re.compile("(Connecting to \S+ \(\S+\)\|(\S+)\|:\d+)")
-        self.filename_extractor = re.compile("(Saving to: [‘'](\S+)[’'])")
 
     def get_document_dependencies(self, document: str) -> List[ExtractedDependency]:
         dependencies = []
+        print(f'Started blocks - {time.time()}')
```

Comment:
Did you want to keep the print statements in here for logging (also ending print stmt on line #50)?

Reply:
Ah, good catch!